src/components/molecules/search-results-list/search-results-list.jsx

```diff
@@ -93,14 +95,16 @@ const SearchResultsList = ({
 					{i18n.results[language]} {fromPage}-{toPage} {i18n.of[language]}{' '}
 					{results.totalResults} {i18n.for[language]}: {keyword}
 				</h4>
-				<SearchResultsPager
-					testid={testIds.RESULTS_PAGER_TOP}
-					current={currentPage}
-					totalResults={results.totalResults}
-					resultsPerPage={resultsPerPage}
-					language={language}
-					keyword={keyword}
-				/>
+				{!hidePage && (
```

Comment:
Can you use `showPage` instead and that shows the pager only when true.

Reply:
So this is the issue - we can't cause best bets and dictionary fails tests cause of async somewhere..